integreat_cms/api/v3/locations.py

```diff
@@ -71,9 +71,24 @@ def transform_poi_translation(poi_translation: POITranslation) -> dict[str, Any]
 
     poi = poi_translation.poi
 
-    primary_contact = Contact.objects.filter(
-        location=poi, area_of_responsibility=""
-    ).first()
+    contacts = Contact.objects.filter(location=poi).all()
+    contact_data = []
+    for contact in contacts:
+        contact_data.append(
+            {
+                "id": contact.id,
+                "area_of_responsibility": contact.area_of_responsibility
+                if contact.area_of_responsibility
+                else None,
+                "name": contact.name,
+                "email": contact.email,
+                "phone_number": contact.phone_number,
+                "website": contact.website,
+            }
+        )
+
+    # for transitional period
+    primary_contact = contacts.first()
```

Comment:
Would it be possible to choose the only contact without an area of responsibility here? (Or none, if all contacts have an AoR, like it was previously?) I assume the app should/wants to display that contact first in the list, if it exists.

Reply:
Yes, good point & exactly

Reply:
Should be fixed now :)

Reply:
Yes, perfect 💪🏼